hack/release.sh

```diff
@@ -23,8 +23,36 @@ website() {
     find website/content/en/${RELEASE_VERSION}/*/*/*.yaml -type f | xargs perl -i -p -e "s/preview/${RELEASE_VERSION}/g;"
 }
 
+editWebsiteConfig() {
+  sed -i '' '/^\/docs\/\*/d' website/static/_redirects
+  echo "/docs/*     	                /${RELEASE_VERSION}/:splat" >>website/static/_redirects
+
+  yq -i ".params.latest_release_version = \"${RELEASE_VERSION}\"" website/config.yaml
+  yq -i ".menu.main[] |=select(.name == \"Docs\") .url = \"${RELEASE_VERSION}\"" website/config.yaml
+
+  editWebsiteVersionsMenu
+}
+
+# editWebsiteVersionsMenu sets relevant releases in the version dropdown menu of the website
+# without increasing the size of the set
+editWebsiteVersionsMenu() {
+  VERSIONS=(${RELEASE_VERSION})
+  while IFS= read -r LINE; do
+    SANITIZED_VERSION=$(echo "${LINE}" | sed -e 's/["-]//g' -e 's/ *//g')
+    VERSIONS+=("${SANITIZED_VERSION}")
+  done < <(yq '.params.versions' website/config.yaml)
+  unset VERSIONS[${#VERSIONS[@]}-2]
```

Comment:
we should also only be including the latest patch version for a minor version in the drop-down

Reply:
Added a TODO to circle back to this.